tests/installation/reboot_after_installation.pm

```diff
@@ -35,11 +35,18 @@ sub run {
         my $svirt = console('svirt');
         $svirt->change_domain_element(os => boot => {dev => 'hd'});
     }
-    wait_screen_change {
-        send_key 'alt-o';    # Reboot
-    };
 
-    power_action('reboot', observe => 1, keepconsole => 1, first_reboot => 1);
+    #use ipmi tools to reboot machine when backend is ipmi
+    if (check_var('BACKEND', 'ipmi')) {
+        power_action('reboot', observe => 1, keepconsole => 1, first_reboot => 1);
```

Comment:
@jerrytt 's purpose is to kill ssh connection must before sending reboot action, since auto-reboot counting was stopped in `installation/await_install` step, we can't use this feature here.
so kill ssh by using `power_action` that is fine, but better call `prepare_system_shutdown` directly, since we do actual reboot in `hard_reset` that using ipmi to reboot machine.

Reply:
Installation is running in remote by ssh .
so you can NOT exit ssh during installation, because you are using ssh for install